### DISCUSSION TITLE
are the bytes in body decrypted?

### DISCUSSION THREAD

**Discussion by H0mTanks** [emanuele-em/proxelar#36] [Q&A] [answered]
Are the bytes displayed in the body field the decrypted TLS bytes?

**Comment by emanuele-em** [comment]
Hi! @H0mTanks Actually not yet! A decoder is surely in our plans, if you want to contribute I will be happy to help you, [async-compression crate](https://docs.rs/async-compression/latest/async_compression/) should be a good start point

**Reply by H0mTanks** [reply]
@emanuele-em Sure I'd love to contribute, I'll take a look at the codebase and the corresponding issue you created. Do we have access to the session keys?, since I assume the proxy is doing the TLS handshake itself.

**Reply by emanuele-em** [reply]
Sure, readme for all infornations

**Comment by emanuele-em** [comment, marked as answer]
#37
